hack/make-rules/verify/shellcheck.sh

```diff
@@ -53,7 +68,18 @@ SHELLCHECK_OPTIONS=(
 )
 
 # actually shellcheck
-docker run \
-  --rm -t -v "${REPO_ROOT}:${REPO_ROOT}" -w "${REPO_ROOT}" \
-  "${SHELLCHECK_IMAGE}" \
-  "${SHELLCHECK_OPTIONS[@]}" "${all_shell_scripts[@]}"
+# tell the user which we've selected and lint all scripts
+# The shellcheck errors are printed to stdout by default, hence they need to be redirected
+# to stderr in order to be well parsed for Junit representation by juLog function
+res=0
+if ${HAVE_SHELLCHECK}; then
+  >&2 echo "Using host shellcheck ${SHELLCHECK_VERSION} binary."
+  shellcheck "${SHELLCHECK_OPTIONS[@]}" "${all_shell_scripts[@]}" >&2 || res=$?
+else
+  >&2 echo "Using shellcheck ${SHELLCHECK_VERSION} docker image."
+  "${DOCKER}" run \
+    --rm -v "${KUBE_ROOT}:${KUBE_ROOT}" -w "${KUBE_ROOT}" \
```

Comment:
I see, last commit fixes this mistake and is conflated with the vagrant helper ... should you fix this here directly, and leave the last commit just for doing the thing that the message says?

Reply:
it's from rebase hell

Reply:
fixed